client/src/components/Modal.js

```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components'
+
+export default function Modal({ handleClose, show, children, recipes }) {
+    return (
+        <ModalDiv block={show ? 'block' : 'none'}>
+            <ContentDiv recipes={recipes}>
+                {children}
+                <button
+                    onClick={handleClose}>
+                    Close
+                    </button>
```

Comment:
That's very picky, but seems like Prettier didn't do its job here :-).

Reply:
Yes, not sure what happened here, but I changed it by hand. :)